tests/test_utils.py

```diff
@@ -1,10 +1,22 @@
 from django.test import TestCase
 
-from djangocms_fil_bootstrap.test_utils.factories import PageVersionFactory
+from djangocms_fil_bootstrap.test_utils.factories import (
+    PageVersionFactory,
+    PageContentWithVersionFactory,
+    UserFactory,
+)
 from djangocms_fil_bootstrap.utils import get_version
+from djangocms_versioning.constants import ARCHIVED
 
 
 class UtilsTestCase(TestCase):
     def test_get_version(self):
         version = PageVersionFactory()
         self.assertEqual(get_version(version.content.page), version)
+
+    def test_get_version_for_multiple_versions(self):
+        user = UserFactory(username="user1")
+        page_content = PageContentWithVersionFactory(version__state=ARCHIVED)
+        version = PageVersionFactory(content__page=page_content.page)
+        latest_version = version.copy(created_by=user)
```

Comment:
I don't think you should be using the copy method for data setup. If the logic in the copy method ever changes then your test data setup may be impacted which is never a good thing. So ideally, create this version with factories.
Another thing worth doing to make this test better though is to use freezegun. For example:
```
with freezetime('2010-10-10'):
    page_content = PageContentWithVersionFactory(version__state=ARCHIVED)
latest_version = PageVersionFactory(content__page=page_content.page)
with freezetime('1999-09-09'):
    PageVersionFactory(content__page=page_content.page, state=UNPUBLISHED)
```
Doing it like this will set the created_date on the version to be in the past. It will also ensure that you're not only doing `latest()` in `get_version`, but that you're doing it on the created or modified date rather than say on `pk`.